app/react/translations/detail/renderers/ArrayRenderer.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+import React from 'react';
+
+type PropTypes = {
+  value: string
+}
+
+export default function ArrayRenderer({ value }: PropTypes) {
```

Comment:
Return value can be typed too, see http://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/
```javascript
export default function ArrayRenderer({ value }: PropTypes): React$Element {}